### PR TITLE
Fix maven without profile

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: server/container/guice/pom.xml [hunk 1]
```diff
@@ -37,21 +37,8 @@
 
     <modules>
         <module>guice-common</module>
+        <module>cassandra-guice</module>
+        <module>memory-guice</module>
     </modules>
-
-    <profiles>
-        <profile>
```
Comment (mbaechler) [inline server/container/guice/pom.xml]: you break benoit's intent by doing this. what is your problem actually ?

Comment (aduprat) [inline server/container/guice/pom.xml]: If you don't give the Cassandra profile to maven, the `SmtpTestModule` (in mpt-smtp-cassandra) won't compile as you will not satisfy the james-server-cassandra-guice dependency.

Comment (mbaechler) [inline server/container/guice/pom.xml]: are these projects' tests disabled at another level then ?

Comment (chibenwa) [inline server/container/guice/pom.xml]: I think you can just activate cassandra smtp tests if cassandra profile is present. Right ?

Comment (aduprat) [inline server/container/guice/pom.xml]: OK

**Comment by mbaechler** [comment]
:+1:

**Comment by rouazana** [comment]
:+1:

**Pull request merged**